packages/test/src/test-workflows.ts

```diff
@@ -41,7 +41,17 @@ test.after.always((t) => {
 test.beforeEach(async (t) => {
   const { contextProvider } = t.context;
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-  const testName = t.title.match(/\S+$/)![0];
+  const trimmedTitle = t.title.match(/\S+$/)![0];
+  // Trust me, you don't want to do this part with a regex.
+  const colonPos = trimmedTitle.indexOf(':');
+  let testName: string;
+  if (colonPos != -1) {
+    testName = trimmedTitle.substr(0, colonPos);
+  } else {
+    testName = trimmedTitle;
+  }
+  const testCase = trimmedTitle.substr(colonPos + 1);
```

Comment:
Instead of all of this you could have just kept everything as is and just used test titles like: `not-replay patched`

Reply:
BAHHHHHHHHHHHHHHHHHHH